src/components/PlayWindow.tsx

```diff
@@ -0,0 +1,57 @@
+import { Box, Button, Flex, Text } from '@chakra-ui/react';
+import React from 'react';
+import { QuestionDataSingle } from '../utils/data';
+
+export default function PlayWindow({
+  login,
+  randomArray,
+  handleOnClickDiv,
+  arrayWithRightAnswer,
+  clickedKey,
+  checkingMode,
+  handleChecking,
+}: {
+  login: string;
+  randomArray: QuestionDataSingle;
+  handleOnClickDiv: (key: string) => void;
+  arrayWithRightAnswer: string[];
+  clickedKey: string[];
+  checkingMode: boolean;
+  handleChecking: () => void;
+}) {
+  console.log(randomArray);
+
+  return (
+    <Box>
+      <Text>{login}, let's play a game</Text>
+      <Box>
+        <Text>{randomArray !== undefined && randomArray.question}:</Text>
+        <Flex wrap='wrap' sx={{ border: '1px solid black', p: '5px' }}>
+          {randomArray !== undefined &&
```

Comment:
!!randomArray &&
bo undefined i null to falsy values

Reply:
w sumie lepiej zastrzec że możesz przekazać tutaj randomArray z jakimiś danymi to wtedy map dla pustej tablicy i tak się nie odpali

Reply:
co masz dokladnie na mysli? jakis przyklad?

Reply:
jeżeli randomArray jest typu Array<any> to [].map nic nie zrobi więc będzie git i nie trzeba sprawdzać czy jest różne od undefined